modules/persistent-storage-csi-ebs-operator-install.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-install_{context}"]
+= Installing the AWS Elastic Block Store CSI Operator
+
+The AWS Elastic Block Store (EBS) Container Storage Interface (CSI) Operator enables the replacement of the existing AWS EBS in-tree storage plug-in.
+
+When you install the AWS EBS CSI Operator, {product-title} provides the CSI driver that allows you to use CSI volumes with the `PersistentVolumeClaims`, `PersistentVolumes`, and `StorageClasses` API objects.
+
+The AWS EBS CSI Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the AWS EBS CSI driver in your cluster.
+
+.Prerequisites
+* Access to the `oc` command-line interface (CLI) or the {product-title} web console.
+
+.Procedure
+. Create a new namespace in which to run the AWS EBS CSI. This can be done from either the CLI or the web console.
++
+* From the CLI:
++
+----
+$ oc new-project csi-ebs
+----
++
+* From the web console:
+.. Select *Projects* -> *Create Project*.
+.. Type the name of the project, such as `csi-ebs`.
+.. Optional: Enter a Display Name and Description.
+.. Click *Create*.
+
+
+. Install the AWS EBS CSI Operator from the web console.
+
+.. Log in to the web console.
+
+.. Navigate to *Operators* -> *OperatorHub*.
+
+.. Type *AWS EBS CSI* into the filter box to locate the AWS EBS CSI Operator.
+
+.. Click *Install*.
+
+.. On the *Create Operator Subscription* page, select *A specific namespace on the cluster*. Select *csi-ebs* from the drop-down menu.
+
+.. Adjust the values for the *Update Channel* and *Approval Strategy* to the desired values.
+
+.. Click *Install*.
+
+. Once finished, the AWS EBS CSI Operator will be listed in the *Installed Operators* section of the web console.
+
+.. Navigate to *Operators* -> *OperatorHub*.
```

Comment:
From this line and the next lines, looks it's duplicated content.

Reply:
> From this line and the next lines, looks it's duplicated content.

Yikes, good catch. I was reusing the workflow from the [LSO install ](https://docs.openshift.com/container-platform/4.2/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local)page. But because CSI EBS isn't on OLM yet, I couldn't test this procedure. :) I've fixed, and filed [BZ 1840399](https://bugzilla.redhat.com/show_bug.cgi?id=1840399) to also address this in the LSO section.